### ISSUE TITLE
docker-compose build issue

### ISSUE THREAD

**Issue by parhamgh2020** [hrezaeianzadeh/django-screen-listing#1]
An error has occurred while attempting to execute the "docker-compose build" command.

error: 
failed to solve: executor failed running [/bin/sh -c apt-get install wget gcc python3 python3-pip python3-psycopg2 -y     && apt-get autoremove -y     && apt-get clean -y]: exit code: 100

The issue lies in creating the image from the Dockerfile at line 9.

**Comment by parhamgh2020** [comment]
The base image used is "ubuntu:22.04", which may not be the most optimized image for the specific requirements of the code

**Comment by parhamgh2020** [comment]
Why are packages like wget and gcc needed?

**Comment by hrezaeianzadeh** [comment]
Yeah I just copied the line from another Dockerfile of mine,  but I can optimize it for you shortly for less packages

**Comment by hrezaeianzadeh** [comment]
> An error has occurred while attempting to execute the "docker-compose build" command.
> 
> error: failed to solve: executor failed running [/bin/sh -c apt-get install wget gcc python3 python3-pip python3-psycopg2 -y && apt-get autoremove -y && apt-get clean -y]: exit code: 100
> 
> The issue lies in creating the image from the Dockerfile at line 9.

but the error you are mentioning is odd because the build process working fine for me
